<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseModuleAnimation</class>
 <widget class="QWidget" name="DatabaseModuleAnimation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>588</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="AnimationViewer" name="m_pAniViewer">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="3" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="m_pAniPlayerButton">
          <property name="text">
           <string>Play</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="m_pFrame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="1" column="0" alignment="Qt::AlignTop">
            <widget class="QLabel" name="m_pCurrentFrameLabel_6">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>index:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" alignment="Qt::AlignTop">
            <widget class="QSpinBox" name="m_pCurrentFrame">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1" alignment="Qt::AlignTop">
            <widget class="QSpinBox" name="m_pFrameTime">
             <property name="maximum">
              <number>999999</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0" alignment="Qt::AlignTop">
            <widget class="QLabel" name="m_pFrameTimeLabel_4">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>time (msec):</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" alignment="Qt::AlignTop">
            <widget class="QLabel" name="m_pCurrentFrameLabel_7">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>current frame:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <widget class="QFrame" name="m_pSpriteFrame">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>141</width>
          <height>211</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QFormLayout" name="formLayout_2">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="1" column="1">
            <widget class="QLabel" name="label_23">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>selcted sprite:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_24">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>x:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="m_pSpritePosX">
             <property name="minimum">
              <number>-999999999</number>
             </property>
             <property name="maximum">
              <number>999999999</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_25">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>y:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="m_pSpritePosY">
             <property name="minimum">
              <number>-999999999</number>
             </property>
             <property name="maximum">
              <number>999999999</number>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>z value:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="m_pSpriteZValue"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_27">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>scale:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QDoubleSpinBox" name="m_pSpriteScale">
             <property name="suffix">
              <string/>
             </property>
             <property name="decimals">
              <number>4</number>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_30">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>opacity:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QDoubleSpinBox" name="m_pSpriteOpacity">
             <property name="decimals">
              <number>4</number>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_29">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>rotation:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QSpinBox" name="m_pSpriteRotation">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>359</number>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_26">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>spriteID:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QSpinBox" name="m_pSpriteSpriteID">
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="maximum">
              <number>999999999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_19">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>animation:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_pGridCheckbox">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>grid</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnimationViewer</class>
   <extends>QGraphicsView</extends>
   <header>animationviewer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
